docs/user_guide/proxy.md

```diff
@@ -0,0 +1,86 @@
+# Use dfdaemon as an HTTP proxy
+
+Dfdaemon can be used as an HTTP proxy to speed up image pulling from any registry
+as well as general HTTP downloads.
+
+Please first ensure that you know how to install and run [supernode](install_server.md)
+and [dfdaemon](install_client.md).
+
+**HTTPS support is currently very limited**. All HTTPS request will be tunneled
+directly, without dfget. If you want to use HTTPS enabled registries, we suggest
+that you use HTTP for your images layers, and HTTPS for other apis like authentication.
```

Comment:
In fact, as far as I know, only `registry.**-**.aliyuncs.com` registries that has done like that, and it is unreasonable for users to modify the image registries.

Reply:
Sure. I removed the registry setting suggestion.